Code/GraphMol/FileParsers/SequenceParsers.cpp

```diff
@@ -1187,6 +1187,7 @@ static const char *LookupHELMPeptideMonomer(const char *ptr) {
       break;
     case 'D':
       if (ptr[1] == '\0') return "ASP";
+      break;

```

Comment:
If ptr[1] is not 0, it falls through to the next case statement, sees that ptr[1] is not 0, and then breaks.

The change is clearly the correct thing to do, but I don't see how it actually fixes anything.